src/entities/activity/ui/items/ActionPlan/Body.tsx

```diff
@@ -0,0 +1,15 @@
+import { PropsWithChildren } from 'react';
+
+import { useXScaledDimension } from './hooks';
+
+import Box from '~/shared/ui/Box';
+
+export const Body = ({ children }: PropsWithChildren) => {
+  const gap = useXScaledDimension(32);
+
+  return (
+    <Box display="flex" flexDirection="column" gap={`${gap}px`} overflow="hidden">
```

Comment:
Just noticed the bottommost edge of the content was really tight against the logo compared to Figma, needs about 40px:
```suggestion
    <Box display="flex" flexDirection="column" gap={`${gap}px`} mb="40px" overflow="hidden">
```